scripts/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# This script is executed by Travis CI for every successful push (on any branch, PR or not).
+# It runs some basic tests on pages. If the build is also a PR, additional
+# checks are run through the check-pr script, and any message or error is sent
+# to tldr-bot to be commented on the PR.
+#
+# NOTE: must be run from the repository root directory to correctly work!
+# NOTE: `set -e` is applied conditionally only if needed.
+
+# Default test.
+function default_test {
+  markdownlint pages*/**/*.md
+  tldr-lint ./pages
+}
+
+# Special test for Travis CI pull request builds.
+# Runs default_test collecting errors for tldr-bot.
+function pr_test {
+  errs=`default_test 2>&1`
+
+  if [ -n "$errs" ]; then
```

Comment:
Does this:

```bash
function f {
    res=`cmd`

    if [ $res != "err" ]; then
        return 0
    fi 

    echo "error!"
    exit 1
}
```

... really read better/clearer than this?

```bash
function f {
    res=`cmd`

    if [ $res = "err" ]; then
        echo "error!"
        exit 1
    fi
}
```

Reply:
Well, this is what it would look like:

```
  if [ -z "$errs" ]; then
   return 0
 fi

  echo -e "Test failed!\n$errs\n" >&2
  echo 'Sending errors to tldr-bot.' >&2
  echo -n "$errs" | python3 scripts/send_to_bot.py error
  exit 1
```

So yeah, I think it is clearer. It is always good to unindent the main case, and indent the shorter, return case.

Reply:
That's not the "main case" though, it's an exception, that's why it seemed unnatural to me.

I mean, it's like:

    if (ok)
        return Ok;
    return Err;

I've never seen this pattern in C, Go, Python or whichever other language.

Reply:
By "main case", I meant the case with the main logic. It can be the error case or the no-error case. And it is very much a pattern in Go I can assure you ;) I am not too strongly attached to it though. So do go ahead with the existing way if you feel more comfortable with it.